src/commands/os/download.ts

```diff
@@ -19,6 +19,7 @@ import { Flags, Args } from '@oclif/core';
 import Command from '../../command';
 import * as cf from '../../utils/common-flags';
 import { stripIndent } from '../../utils/lazy';
+import { isESR } from '../../utils/image-manager';
```

Comment:
Let's make this an async import (exactly where we had the `await import('balena-image-manager');`), so that node only loads the "image-manager" code when we are in a command that actually uses it.